src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -1307,7 +1307,7 @@ void ExceptionTracker::InitializeCrawlFrameForExplicitFrame(CrawlFrame* pcfThisF
     }
     CONTRACTL_END;
 
-    INDEBUG(memset(pcfThisFrame, 0xCC, sizeof(*pcfThisFrame)));
+    INDEBUG(*pcfThisFrame = {});
 
```

Comment:
```suggestion
```
This can be deleted. Zero initialization won't be very effective in flagging uninitialized fields that the original tried to do.